torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -422,6 +417,36 @@ def from_hp_to_fpx(
         tensor_impl = tensor_impl_ctr(floatx_packed, scale, None, _layout)
         return cls(tensor_impl, block_size, original_shape, dtype=input_float.dtype)
 
+    @classmethod
+    def from_hp_to_float8(
+        cls,
+        input_float: torch.Tensor,
+        target_dtype: torch.dtype,
+        block_size: Tuple[int, ...],
+        _layout: Layout = PlainLayout(),
+    ):
```

Comment:
a docblock here should explain the difference between `from_hp_to_floatx`, `from_hp_to_fpx`, `from_hp_to_float8`